src/webui/www/private/newfeed.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="${LANG}">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>QBT_TR(Please type a RSS feed URL)QBT_TR[CONTEXT=RSSWidget]</title>
+    <link rel="stylesheet" href="css/style.css?v=${CACHEID}" type="text/css" />
+    <script src="scripts/lib/mootools-1.2-core-yc.js"></script>
+    <script src="scripts/lib/mootools-1.2-more.js"></script>
+    <script src="scripts/misc.js?locale=${LANG}&v=${CACHEID}"></script>
+    <script>
+        'use strict';
+
+        new Keyboard({
+            defaultEventType: 'keydown',
+            events: {
+                'Enter': function(event) {
+                    $('submitButton').click();
+                    event.preventDefault();
+                },
+                'Escape': function(event) {
+                    window.parent.closeWindows();
+                    event.preventDefault();
+                },
+                'Esc': function(event) {
+                    window.parent.closeWindows();
+                    event.preventDefault();
+                }
+            }
+        }).activate();
+        window.addEvent('domready', function() {
+            $('feedURL').focus();
+            const path = decodeURIComponent(new URI().getData('path'));
+            $('submitButton').addEvent('click', function(e) {
+                new Event(e).stop();
+                // check field
+                const feedURL = $('feedURL').value.trim();
+                if (feedURL === '') {
+                    alert('QBT_TR(Name cannot be empty)QBT_TR[CONTEXT=HttpServer]');
```

Comment:
Is `HttpServer` the right context?

Reply:
The string is already used in `rename_file.html` and so already translated to some languages.